src/Umbraco.Core/Configuration/GlobalSettings.cs

```diff
@@ -391,5 +391,28 @@ public bool UseHttps
                 }
             }
         }
+
+
+        /// <summary>
+        /// An int value representing the time in milliseconds to lock the database for a write operation
+        /// </summary>
+        /// <remarks>
+        /// The default value is 1800 milliseconds
+        /// </remarks>
+        /// <value>The timeout in milliseconds.</value>
+        public int SqlWriteLockTimeOut
+        {
+            get
+            {
+                try
+                {
+                    return int.Parse(ConfigurationManager.AppSettings[Constants.AppSettings.SqlWriteLockTimeOut]);
+                }
+                catch
+                {
+                    return 1800;
```

Comment:
Default should be changed to perhaps 5000.

Reply:
I kept it the same so that it will not change any behavior for now unless specifically configured.